canonical-web-and-design/cn.ubuntu.com
# Add secondary nav and new /internet-of-things/core page

## Done

- Add secondary nav for IoT section
- Add new core page at `/internet-of-things/core` - refactored this [branch](https://github.com/zprood/cn.ubuntu.com/blob/master/templates/core/index.html) as the page wasn't responsive and the headings weren't ordered correctly. Also extracted the styles from the html file and injected them into scss files.

## QA

- Visit page at: https://cn-ubuntu-com-496.demos.haus/internet-of-things
- Check secondary nav for IoT - described [here](https://github.com/canonical-web-and-design/web-squad/issues/3890)
- `/internet-of-things/core page` content should match ubuntu.com/core - copy [here](https://github.com/zprood/cn.ubuntu.com/blob/master/templates/core/index.html)

## Issue / Card

[#3890](https://github.com/canonical-web-and-design/web-squad/issues/3890)

## Screenshots

![image](https://user-images.githubusercontent.com/58959073/110113103-85584900-7daa-11eb-9156-f1c721bd3b30.png)
